packages/ses/index.d.ts

```diff
@@ -1,52 +1,133 @@
-/* eslint-disable */
-/**
- * Transitively freeze an object.
- */
-import type { Hardener } from './src/make-hardener';
-import type { CompartmentConstructor } from './src/compartment-shim';
-import type { Lockdown } from './src/lockdown-shim';
-import type { StaticModuleRecord } from './module-shim';
-
-// For scripts.
-declare var harden: Hardener;
-declare var lockdown: Lockdown;
-declare var Compartment : ReturnType<CompartmentConstructor>;
-declare var StaticModuleRecord : StaticModuleRecord;
-
-declare type StaticModuleType = RedirectStaticModuleInterface | FinalStaticModuleType;
-declare interface RedirectStaticModuleInterface {
+
+export type Harden = <T>(value: T) => T;
```

Comment:
I’m going to run with `Harden` because it wasn’t exported previously, I’d rather not `Lockdowner` for consistency, and there won’t be a follow-up just for that change.